### ISSUE TITLE
get next/previous tweets

### ISSUE THREAD

**Issue by shahmohamadi** [Jefferson-Henrique/GetOldTweets-python#231]
I searched a query by this API and get result from different twitter users. how can I get next/previous for example 5 tweets of a specific result by a certain user? Is it possible to get next/previous tweet addresses by given URL?

**Comment by marquisvictor** [comment]
I don't seem to parse your question correctly

**Comment by marquisvictor** [comment]
I think what you're asking is how to get tweets from a particular users timeline?? 

If that is the case, and the querysearch isn't working, i advise you try the twitter API.

**Comment by shahmohamadi** [comment]
> I think what you're asking is how to get tweets from a particular users timeline??
> 
> If that is the case, and the query search isn't working, i advise you try the twitter API.

Hi, no I don't want entire timeline. for example this is part of a specific user timeline:

![photo_2019-06-30_12-33-45](https://user-images.githubusercontent.com/5832955/60394075-b2fbe200-9b33-11e9-9551-244f24cfd9f1.jpg)

and I get just tweet number 2 in my query search results. how can I get tweets number 3,4,5 (next tweets) or tweets number 1 (previous tweet) by having url of tweet 2?

**Comment by marquisvictor** [comment]
yeah i get your question now.. I have no idea how you can go about that. however, you could also query the users timeline, using the username @WulfyDAT, then you select the tweets you need and discard the rest. 
that should also work. 

you can use the command line function 

py3.6 getoldtweets3 --username WulfyDAT --maxtweets 50000
